pkg/controller/handlers/mcpcatalog/mcpcatalog.go

```diff
@@ -248,142 +246,30 @@ func (h *Handler) readMCPCatalogDirectory(catalog string) ([]types.MCPServerCata
 	return entries, nil
 }
 
-// DeleteUnauthorizedMCPServers deletes all MCP servers that are no longer authorized to run.
-// This can happen when a user has launched an MCP server that they used to have permission for,
-// but their access to the catalog that the server came from has been revoked.
-func (h *Handler) DeleteUnauthorizedMCPServers(req router.Request, _ router.Response) error {
```

Comment:
I will reintroduce this functionality in a follow-up.

Reply:
Ultimately, I'm fine with you reintroducing this functionality in a follow up PR. However, that will mean that we'll have a bunch of Kubernetes deployments in main that will have to be manually deleted.